parser/src/lib.rs

```diff
@@ -485,12 +485,12 @@ namespace N(2);
     fn parse_impl() {
         let input = r#"
     impl<T> Iterator<ArrayIterator<T>, T> {
-        next: |it| if it.pos >= 7 { (it, none) } else { some((increment(it), it.arr[it.pos])) },
```

Comment:
Is this merged to main? It uses member access notation we don't support yet. Also I think this should stay as `.`.

Reply:
Yes, but it was just a test on the parser. Anyway, I fixed it.